content/warnings/invalid-aria-prop.md

```diff
@@ -4,8 +4,8 @@ layout: single
 permalink: warnings/invalid-aria-prop.html
 ---
 
-The invalid-aria-prop warning will fire if you attempt to render a DOM element with an aria-* prop that does not exist in the Web Accessibility Initiative (WAI) Accessible Rich Internet Application (ARIA) [specification](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties).
+تحذير 'invalid-aria-prop' سَيُطلَق إذا قُمت بمُحاولة تصيير عنصر DOM مع خاصية '*-aria' غير موجودة في [مواصفات](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties) مبادرة إتاحة الويب "Web Accessibility Initiative WAI" و تطبيق انترنت متاح غني "Accessible Rich Internet Application ARIA".
```

Comment:
```suggestion
تحذير 'invalid-aria-prop' سَيُطلَق إذا قُمت بمُحاولة تصيير عنصر DOM مع خاصية '*-aria' غير موجودة في [مواصفات](https://www.w3.org/TR/wai-aria-1.1/#states_and_properties) "Web Accessibility Initiative WAI" و "Accessible Rich Internet Application ARIA".
```

Reply:
@Fcmam5 could you take a look at this? Thanks :)

Reply:
That's better as you translated it, like we translate to words/names if it's possible and we add the original name between parentheses. Good job!